spring-auto-restdocs-core/src/main/java/capital/scalable/restdocs/payload/JacksonRequestFieldSnippet.java

```diff
@@ -28,13 +26,27 @@
 public class JacksonRequestFieldSnippet extends AbstractJacksonFieldSnippet {
 
     public static final String REQUEST_FIELDS = "request-fields";
+    private Class<?> requestBodyType;
```

Comment:
You can use `Type` instead of `Class<?>` because it's the super class and it's also the return type of the `getType` method.